datastore/src/main/java/com/ds/datastore/BookController.java

```diff
@@ -43,6 +46,7 @@ public BookController(BookRepository repository, BookModelAssembler assembler, B
             book.setStoreID(storeID);
             book.setStore(store);
             EntityModel<Book> entityModel = assembler.toModel(repository.save(book));
+            logger.info("Book has been added");
```

Comment:
I think it would be nice to include information about the book, using variable substitution like here: http://www.slf4j.org/api/org/slf4j/Logger.html